AU2/plugins/custom_plugins/SRCFPlugin.py

```diff
@@ -544,12 +573,42 @@ def answer_publish_pages(self, htmlResponse) -> List[HTMLComponent]:
                 os.remove(localpath)
 
             self._publish_databases(sftp)
+            automatic_backup = self._autobackup(sftp)
 
-        return [Label("[SRCF Plugin] Successfuly published locally generated pages and uploaded database.")]
+        return [
+            Label("[SRCF Plugin] Successfuly published locally generated pages and uploaded database."),
+            Label(f"[SRCF Plugin] Automatically created backup {automatic_backup}")
+        ]
+
+    def _backup_to_remote(self, sftp, backup_name: str):
+        backup_path = REMOTE_BACKUP_LOCATION + "/" + backup_name

```

Comment:
Change `REMOTE_BACKUP_LOCATION` to a pathlib path

```suggestion
        backup_path = REMOTE_BACKUP_LOCATION / backup_name

```